src/combinatorics/quiver/path.lean

```diff
@@ -59,6 +65,28 @@ def comp {a b : V} : Π {c}, path a b → path b c → path a c
 | c p q path.nil := rfl
 | d p q (path.cons r e) := by rw [comp_cons, comp_cons, comp_cons, comp_assoc]
 
+/-- Turn a path into a list. The list contains `a` at its head, but not `b` a priori. -/
+@[simp] def to_list : Π {b : V}, path a b → list V
+| b nil := []
+| b (@cons _ _ _ c _ p f) := c :: p.to_list
```

Comment:
What causes the need for the `@` here? The pattern matching above seems to work fine with `path.cons`.

Reply:
This is because I need to name `c`. The pattern matching above only uses the new morphism, not its source.